hooks/post-command

```diff
@@ -27,6 +27,10 @@ if [ "${BUILDKITE_PLUGIN_CODECOV_FAIL_JOB_ON_ERROR:-${default_fail_job_on_error}
     codecov_args+=(--nonZero)
 fi
 
+if [[ -v BUILDKITE_PLUGIN_CODECOV_FLAGS ]]; then
+    codecov_args+=(--flags="${BUILDKITE_PLUGIN_CODECOV_FLAGS:-}")
```

Comment:
:- means "if the previous var is unset, replace it with whatever comes after :-"
and since we know it's set, I'm not sure it's worth keeping around; just a bare "${THING}" should suffice

Reply:
Thanks, updated